GitCommands/Settings/AppSettings.cs

```diff
@@ -1490,6 +1486,39 @@ public static string GetString(string name, string defaultValue)
             return SettingsContainer.GetString(name, defaultValue);
         }
 
+        private static void LoadEncodings()
+        {
+            Action<Encoding> addEncoding = delegate (Encoding e) { AvailableEncodings[e.HeaderName] = e; };
+
+            string[] availableEncodings = GetString("AvailableEncodings", "").Split(';');
+            if (availableEncodings == null || availableEncodings.Length == 0 || string.IsNullOrWhiteSpace(availableEncodings[0]))
+            {
+                // Default encoding set
+                addEncoding(Encoding.Default);
+                addEncoding(new ASCIIEncoding());
+                addEncoding(new UnicodeEncoding());
+                addEncoding(new UTF7Encoding());
+                addEncoding(new UTF8Encoding(false));
+                addEncoding(Encoding.GetEncoding("CP852"));
```

Comment:
CP852 appear in that commit (https://github.com/schadin/gitextensions/commit/691bf3556bebb4f98b7bbba262ec29761026d27f) because someone needs specific DOS encoding. I think better remove this item (because user can manually add it with your changes) or add instead default DOS encoding (OEMCodePage).